formatter/fmt

```diff
@@ -37,13 +37,14 @@ else
   echo 'Done formatting java'
 fi
 
-echo 'Start format TS'
+echo 'Start formatting with prettier'
 npx prettier \
   --write \
   --config /.prettierrc.js \
   --ignore-path /.prettierignore \
-  {server,browser-test}
-echo 'Done formatting TS'
+  --ignore-unknown \
+  .
```

Comment:
question (blocking): why `.` and not `{server,browser-test}`?

Reply:
It's also formatting a bunch of other file formats, including css, yaml, and markdown.

The full list is here: https://prettier.io/docs/en/index.html